### DISCUSSION TITLE
Welcome to vue-curveball-microservice-monorepo Discussions!

### DISCUSSION THREAD

**Discussion by Nicholaiii** [Nicholaiii/vue-curveball-microservice-monorepo#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NikoDyring** [comment]
Hello. I'm so hyped to be here. :)

Den tir. 5. jan. 2021 kl. 18.10 skrev Nicholai Nissen <
notifications@github.com>:

> 👋 Welcome!
>
> We’re using Discussions as a place to connect with other members of our
> community. We hope that you:
>
>    - Ask questions you’re wondering about.
>    - Share ideas.
>    - Engage with other community members.
>    - Welcome others and are open-minded. Remember that this is a
>    community we
>    build together 💪.
>
> To get started, comment below with an introduction of yourself and tell us
> about what you do with this community.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Nicholaiii/vue-curveball-microservice-monorepo/discussions/1>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACHTPL3ZUEWKNBJWFMRKS2TSYNBZLANCNFSM4VVT4W4A>
> .
>


-- 
[image: photo]
*Nikolaj Dyring Jensen*
Software Engineer

+45 23954072 <+45+23954072>
<http://us.linkedin.com/in/nikolaj-dyring-jensen>
<http://github.com/Nikodyring> <http://twitter.com/nikodyring>
